src/librustc_resolve/lib.rs

```diff
@@ -1969,10 +1996,27 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                 Err(())
             }
         } else {
+
+            // find possible candidates
+            let trait_name = trait_path.segments.last().unwrap().identifier.name;
+            let candidates: Vec<_> =
+                self.lookup_suggestions(
+                    trait_name,
+                    LookupTarget::AllCrates,
+                    Namespace::TypeNS,
+                    move |def: Def|
```

Comment:
Is the `move` keyword is required here? (I'd be surprised, given that there is no state)